src/blocks/Gassim/motion.js

```diff
@@ -0,0 +1,73 @@
+window.onload = function() {
+    var pos = 0;
+    var pwin = window.innerWidth/2;
+    var pos2 = 0;
+    var pwin2 = window.innerHeight/3;
+    var pos3 = window.innerWidth/6;
+    var pwin3 = window.innerWidth/2;
+    var pos4 = window.innerWidth/6;
+    var pwin4 = window.innerHeight/3;
+    var windowWidth = (window.innerWidth/2)-window.innerWidth/2%1;
+    var windowHeight = (window.innerHeight/2)-window.innerHeight/2%1;
+    var b = 0;
+    var keyword = "nature,greenscreen";
+    var images = document.getElementsByClassName('images');
+    var t = setInterval(move, 10);
+    var t = setInterval(insert, 50);
+    var index = 1;
+    
+    window.addEventListener('resize', checkWindowSize)
+    function checkWindowSize() {
+     windowWidth = (window.innerWidth/2)-window.innerWidth/2%1;
+     windowHeight = (window.innerHeight/2)-window.innerHeight/2%1;
+     pwin = window.innerWidth/2;
+    pwin2 = window.innerHeight/3;
+    pwin3 = window.innerWidth/2;
+    pwin4 = window.innerHeight/3;
+    };
+
+    function insert() {
+        if(keyword == "nature,greenscreen"){keyword="male,abs"}
+        else if (keyword == "male,abs") {keyword="male,shirtless"}
+        else if (keyword == "male,shirtless") {keyword= "nature,greenscreen"}
+    }
+
+
+    
+    function move() {
+        images[b].innerHTML = `<img src="https://source.unsplash.com/random/${windowWidth}x${windowHeight}/?${keyword};SameSite=Lax"/>`
```

Comment:
A cleaner solution would be to create a image element with document.createElement instead of a innerHTML string which should be avoided.
Can you change this one?
I just want you to get the best out of yourself.
If you create the img element with document.createElement in a seperat function, then you will be able to reuse this part more easily @gassims

Reply:
Hi!

Thank you so much for the support!

innerHTML changed the HTML code itself so the API requests were cleaner and consistent since each time it was 'new' so using document.createElement alone wasn't enough; however, I created a function to replace the code and added a fix to avoid using innerHTML. I know the code needs a lot more work and that you pointed out one which is necessary. Thanks!